bigchaindb/backend/mongodb/query.py

```diff
@@ -153,13 +153,17 @@ def get_spent(conn, transaction_id, output):
     cursor = conn.run(
         conn.collection('bigchain').aggregate([
             {'$match': {
-                'block.transactions.inputs.fulfills.txid': transaction_id,
-                'block.transactions.inputs.fulfills.output': output
+                'block.transactions.inputs.fulfills': {
```

Comment:
I wonder what it would happen if you write a transaction to the database with a fulfills key that instead of:
```json
{
    "txid": "...",
    "output": 0
}
```
is:
```json
{
    "output": 0,
    "txid": "..."
}
```